python/langsmith/evaluation/_arunner.py

```diff
@@ -81,6 +83,8 @@ async def aevaluate(
         description (Optional[str]): A description of the experiment.
         max_concurrency (Optional[int]): The maximum number of concurrent
             evaluations to run. Defaults to None.
+        num_repetitions (int): The number of times to repeat the evaluation.
```

Comment:
ha sorry - maybe "the number of times to run the evaluation" since "to repeat" makes 1 sound like we would run twice